docker/lb/README.md

```diff
@@ -0,0 +1,27 @@
+# nginx based Load Balancer for Avalon
+
+This shows how to create nginx based Load Balancer to forward the requests
+coming from clients to one of the many avalon-listeners running in backend.
+
+## Running
+
+1. Build and run Avalon components.
+    ```bash
+    docker-compose -f docker/lb/docker-compose-lb.yaml up --build
+
+    The above command is going to start nginx based service (avalon-lb) along
+    with other avalon services. The client is going to submit the transaction
+    to avalon-lb Load Balancer and this load balancer is going to forward the
+    requests to 3 instances of avalon-listeners in the backend in round-robin
+    fashion.
```

Comment:
Add where the load balancing is configured (that is in ngix.conf).
Also add that ngix listens to port 80 and forwards requests to one of the backend servers.
I would also use another port besides 80 for an example, as 80 is commonly used for the web.
For example port 9947
 So change the generic_client.py line to use http://avalon-lb:9947 and change 80 to 9947 in nginx.conf